roles/edpm_nova/tasks/adoption.yml

```diff
@@ -0,0 +1,30 @@
+---
+
+- name: Verify and stop nova_virtlogd
+  become: true
+  tags:
+    - adoption
+    - nova
+  block:
+    - name: Check if nova_virtlogd container exists
+      ansible.builtin.command: podman ps -a --filter name=nova_virtlogd --format \{\{.Names\}\}
+      register: nova_virtlogd_status
+      changed_when: false
+    - name: Stop nova_virtlogd containers not managed by service units
+      ansible.builtin.shell: "podman stop nova_virtlogd '||' true"
```

Comment:
Does podman return a nonzero exitcode if it succeeds? Do we need to ignore all the exit codes?